css/main.css

```diff
@@ -1,9 +1,9 @@
 body {
     margin: 0;
     padding: 20px 40px;
-    
-    font-family: 'Open Sans', 'Helvetica', sans-serif;
-    
+
+    font-family: "Lato", sans-serif;
```

Comment:
This is highly subjective. Saying "–– Lato and montserrat are superior fonts." is just an opinion

Reply:
I mean it is highly subjective but they do objectively look better and clearer, and using the 2 of them makes clear difference between different font-sizes and text objects of different importance.